content/en/cloudcraft/account-management/enable-sso.md

```diff
@@ -4,51 +4,53 @@ title: Enable SSO
+  accounts.
+- **Google Workspace SSO**: Google SSO does not require any additional setup, select **Sign in with Google** on Cloudcraft's signup or login pages.
+- **SAML SSO**: Available for Cloudcraft Pro and Enterprise accounts, SAML SSO federates with your organization's existing identity provider, allowing your users to log in with their existing accounts and for your organization to centrally manage who has access to the application.
 
-This article is all about SAML Enterprise SSO and how to set it up in your account.
+This article is all about SAML SSO and how to set it up in your account.
 
 ## Setting up SAML/SSO
 
-<div class="alert alert-info">The SAML Enterprise SSO feature is only available for the Enterprise plan, and can only be configured by the account owner.
-</div>
+<div class="alert alert-info">Only the account owner configures the SAML SSO feature. If the account owner is unable to configure SSO, <a href="https://app.cloudcraft.co/app/support" title="Contact the Cloudcraft support team">contact the Cloudcraft support team</a> to enable this feature.</div>
 
-1. Navigate to **User** > **Security & SSO**.
-2. Next, register Cloudcraft as a new application with your SAML identity provider. For detailed instructions, see the following articles:
-    - [Enable SSO With Azure AD][1]
-    - [Enable SSO With Okta][2]
-    - [Enable SSO With a Generic Identity Provider][3]
-3. The details you need to create a new application with your identity provider can be found in the same modal window.
+1. Go to **User** > **Security & SSO**.
+2. Register Cloudcraft as a new application with your SAML identity provider. For detailed instructions, see the following articles:
+    - [Enable SSO With Azure AD][2]
+    - [Enable SSO With Okta][3]
+    - [Enable SSO With a Generic Identity Provider][4]
+3. Find the necessary details to create a new application with the identity provider in the same window.
 
-{{< img src="cloudcraft/account-management/enable-sso/service-provider-details.png" alt="Image provider metadata modal" responsive="true" style="width:100%;">}}
+{{< img src="cloudcraft/account-management/enable-sso/service-provider-details.png" alt="Cloudcraft SAML service provider integration settings." responsive="true" style="width:100%;">}}
 
-4. After creating the application, navigate back to Cloudcraft and upload the metadata file from your identity provider.
-5. Toggle the **SAML Single Sign-On is enabled** option.
-6. If you prefer to have your users access Cloudcraft only via your identity provider, enable the **Strict mode** option.
+4. After creating the application, return to Cloudcraft and upload the metadata file from the identity provider.
+5. Select the **SAML Single Sign-On is enabled** option.
+6. Enable the **Strict mode** option if restricting access to Cloudcraft to only SAML SSO users is required.
```

Comment:
Is this sentence trying to say that if you are restricting access to Cloudcraft to only SAML SSO users, you must enable Strict mode?

```suggestion
6. Enable the **Strict mode** option if restricting access to Cloudcraft to only SAML SSO users is required.
```

Reply:
Yes, something like that. To restrict access to only SAML SSO, you need to enable strict mode.

Reply:
Thanks @jamesponddotco ! I made the suggestion below.